src/common/LoadingIndicator.js

```diff
@@ -48,19 +55,26 @@ export default class LoadingIndicator extends PureComponent {
   };
 
   render() {
-    const { active, caughtUp, size } = this.props;
+    const { active, caughtUp, size, backgroundColor } = this.props;
```

Comment:
Can we do it with `backgroundColor: 'transparent'`

Reply:
Tried with `backgroundColor: 'transparent'`. Didn't worked. 
<img width="96" alt="screen shot 2017-09-23 at 10 02 13 pm" src="https://user-images.githubusercontent.com/18511177/30775080-0c2dc256-a0ab-11e7-9121-6b1244d8826f.png">


Basically I am applying background color to hide semi circle.

Reply:
... but then we don't get the spinning effect?